docs/experiments-trec2018.md

```diff
@@ -0,0 +1,126 @@
+      >& log.cw12lite2013.pos+docvectors+rawdocs &
+      ```
+      about 8.3GB of the index
+    - If you would like to build the index from scratch (e.g. you do not have clueweb12 full index):
+      ```
+      nohup sh Anserini/target/appassembler/bin/IndexCollection -collection ClueWeb12Collection \
+      -generator JsoupGenerator -threads 44 -input /path/to/cw12 -index \
+      lucene-index.cw12lite2013.pos+docvectors+rawdocs -storePositions -storeDocvectors -storeRawDocs \
+      -whitelist Anserini-data/TREC2018/CENTRE/task2/cw12lite2013_docids >& log.cw12lite2013.pos+docvectors+rawdocs &
+      ```
+  - index of CW12Lite2018:
+    - If you already have the ClubWeb12 Full index with `-storeRawDocs` option enabled when indexing:
+      ```
+      nohup Anserini/target/appassembler/bin/IndexUtils -index lucene-index.cw12.pos+docvectors+rawdocs \
+      -dumpRawDocs Anserini-data/TREC2018/CENTRE/task2/cw12lite2018_docids &
+      ```
+      about 28GB in raw size
+      ```
+      nohup Anserini/target/appassembler/bin/IndexCollection -collection HtmlCollection \
+      -input cw12lite2018_docids_rawdocs.dump -index lucene-index.cw12lite2018.pos+docvectors+rawdocs \
+      -generator JsoupGenerator -threads 44 -storePositions -storeDocvectors -storeRawDocs \
+      -optimize >& log.cw12lite2018.pos+docvectors+rawdocs &
+      ```
+      about 12GB of the index
+    - If you would like to build the index from scratch (e.g. you do not have clueweb12 full index):
+      ```
+      nohup sh Anserini/target/appassembler/bin/IndexCollection -collection ClueWeb12Collection \
+      -generator JsoupGenerator -threads 44 -input /path/to/cw12 -index \
+      lucene-index.cw12lite2018.pos+docvectors+rawdocs -storePositions -storeDocvectors -storeRawDocs \
+      -whitelist Anserini-data/TREC2018/CENTRE/task2/cw12lite2013_docids >& log.cw12lite2018.pos+docvectors+rawdocs &
+      ```
+  - index of snippets 2013
+    ```
+    sh Anserini/target/appassembler/bin/IndexCollection -input \
+    Anserini-data/TREC2018/CENTRE/task2/searchengine_snippets/snippets2013_anserini/ \
+    -collection JsonCollection -index lucene-index.snippets2013.pos+docvectors+rawdocs \
+    -generator JsoupGenerator -threads 8 -uniqueDocid -storePositions -storeDocvectors \
+    -storeRawDocs -optimize >& log.snippets2013.pos+docvectors+rawdocs
+    ```
+  - index of snippets 2018
+    ```
+    Anserini/target/appassembler/bin/IndexCollection -input \
+    Anserini-data/TREC2018/CENTRE/task2/searchengine_snippets/snippets2018_anserini/ \
+    -collection JsonCollection -index lucene-index.snippets2018.pos+docvectors+rawdocs \
+    -generator JsoupGenerator -threads 8 -uniqueDocid -storePositions -storeDocvectors \
+    -storeRawDocs -optimize >& log.snippets2018.pos+docvectors+rawdocs
+    ```
+#### (Optional) Get Wikipedia Dump:
++ `curl -O https://dumps.wikimedia.org/enwiki/20180620/enwiki-20180620-pages-articles.xml.bz2`
```

Comment:
Can you remember to save a copy of this dump somewhere?
Old Wikipedia dumps get removed after a while...

Reply:
okay